dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js

```diff
@@ -1,9 +1,43 @@
 import { disableCMP } from '../../lib/disableCMP';
 import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
-
+import { config } from '../../../fixtures/config';
```

Comment:
Importing config from fixtures, so that I can explicitly set a switch

Reply:
Perhaps instead of altering the the data during the load, we can switch to using POST requests and send the switch set as we'd intend for each test,

e.g 
```ts
cy.visit(`/Article`, {
	method: 'POST',
	body: JSON.stringify(articleWithSigninGate)
})
```

Reply:
Just updating this, we can use the existing generated fixtures for this:

```ts
import { Standard } from '../../../fixtures/generated/articles/Standard'

const article = {
	..Standard,
		config: {
			...Standard.config,
			switches: {
				personaliseSignInAfterCheckout: true,
				...Standard.switches
			}
		}
	}

cy.visit(`/Article`, {
	method: 'POST',
	body: JSON.stringify(article)
})
```
I think since the fixtures are in typescript, we may want to check if the cypress config allows us to import TS files

Reply:
Thanks, that's working well 👍